extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -141,7 +141,7 @@ export class AmpGeo extends AMP.BaseElement {
     // match to \w+ to prevent xss vector
     if (getMode(this.win).geoOverride &&
       (isCanary(this.win) || getMode(this.win).localDev) &&
-      getMode(this.win).geoOverride.match(/\w+/)) {
+      getMode(this.win).geoOverride.match(/^\w+$/)) {
```

Comment:
`/^\w+$/.test(getMode(this.win).geoOverride)` is faster (not capture created) and a bit more readable.

I'd consider adding this test at the end of the function and neutralizing `this.country` if it fails.

Reply:
`this.country_` has a default value it picked up from the SCS hotpatch so I'm testing before I overwrite it.  Will make the regex change.